data_processing.py

```diff
@@ -0,0 +1,128 @@
+import os
+import pandas as pd
+import torch
+from torch.utils.data import (DataLoader, RandomSampler, SequentialSampler, TensorDataset)
+from collections import namedtuple
+
+class LeftRightProcessor: 
```

Comment:
Why does it have this name? Is there another name that would make its function clearer?